.gitignore

```
@@ -134,4 +134,6 @@ _thumbnails/
 content/pages/communications.md
 content/pages/contributing.md
 content/pages/code_of_conduct.md
-content/notebooks_gallery/
+content/pages/links.md
+content/pages/links/
```

Change this to only exclude markdown files. 

```suggestion
content/pages/links/*.md
```